source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -1366,27 +1377,29 @@ let from_json_list source_json_list =
   >>= fun implicit_sinks ->
   let parse_integer_option name =
     let parse_single_json (path, json) =
-      match member "options" json with
-      | `Null -> Result.Ok None
+      match Json.Util.member_nullable "options" json with
+      | { JsonParsing.Node.value = `Null; _ } -> Result.Ok None
       | options -> (
-          match member name options with
-          | `Null -> Result.Ok None
-          | `Int value -> Result.Ok (Some value)
-          | json ->
+          try
+            let options_node = Json.Util.member name options in
+            let options_value = Json.Util.to_int_option options_node in
+            match options_value with
+            | None -> Result.Ok None
+            | Some value -> Result.Ok (Some value)
+          with
+          | Json.TypeError { message; json } ->
               Error
                 (Error.create
                    ~path
-                   ~kind:
-                     (Error.UnexpectedJsonType
-                        { json; message = "Expected integer, got"; section = Some "options" })))
+                   ~kind:(Error.UnexpectedJsonType { json; message; section = Some "options" })))
     in
     List.map source_json_list ~f:parse_single_json
     |> Result.combine_errors
     >>| List.filter_map ~f:Fn.id
     >>= function
     | [] -> Result.Ok None
     | [value] -> Result.Ok (Some value)
-    | _ -> Result.Error [{ Error.path = None; kind = Error.OptionDuplicate name }]
+    | _ -> Result.Error [{ Error.path = None; kind = Error.OptionDuplicate name; location = None }]
```

Comment:
Maybe in another PR, we should provide the location of the first duplicate. Same below.

Reply:
Yes - wanted to not overwhelm this PR with a lot of changes